torch/csrc/distributed/autograd/rpc_messages/cleanup_autograd_context_resp.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include <torch/csrc/distributed/rpc/message.h>
+#include <torch/csrc/distributed/rpc/rpc_command_base.h>
+#include <vector>
+
+namespace torch {
+namespace distributed {
+namespace autograd {
+
+// Used to request other workers to clean up their autograd context.
```

Comment:
Mention that this is an empty response for the `CleanupAutogradContextReq`. The current comment seems to indicate this call is responsible for cleaning up the context.